### PR TITLE
Add port to docker run call and document RUST_LOG

### PR DESCRIPTION
**Pull request by znkr**
I was a little confused when I set this up and needed to invest some time to figure this out. I hope this can help other users that run into similar problems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ You can run it using the following example and pass configuration environment va
 $ docker run \
   -e 'EXPORTER_USERNAME=your-username@acme.tld' \
   -e 'EXPORTER_PASSWORD=your-password' \
+  -p '9898:9898'
```
Comment (eko) [inline README.md:39]: Could you add a `\` at the end of the line please?

**Review by eko — COMMENTED** [review]
Thank you for these additions!

File: README.md (hunk 1 above)
Comment (znkr) [reply README.md]: Nice catch, done. :)

**Review by eko — APPROVED** [review]
Great, thank you again :)

**Pull request merged**